src/TrackProgressInternal/SFA.DAS.TrackProgressInternal.Api.AcceptanceTests/ApiFixture.cs

```diff
@@ -0,0 +1,34 @@
+﻿using AutoFixture;
+using SFA.DAS.TrackProgressInternal.Api.AcceptanceTests;
+
+namespace SFA.DAS.TrackProgressInternal.Tests;
+
+public class ApiFixture
+{
+    protected ApiFactory factory = null!;
+    protected Fixture fixture = null!;
+    protected HttpClient client = null!;
```

Comment:
For protected types, resharper is  suggesting they should be capitalised ie Factory, Fixture... etc